src/utilities.jl

```diff
@@ -154,10 +154,10 @@ feature with `ClimaCore.enable_threading() = false`.
 function precomputed_quantities!(Y, p, t, colidx)
     ᶜuₕ = Y.c.uₕ
     ᶠw = Y.f.w
-    (; ᶜuvw, ᶜK, ᶜts, ᶜp, params, thermo_dispatcher) = p
+    (; ᶜū, ᶜK, ᶜts, ᶜp, params, thermo_dispatcher) = p
```

Comment:
does this need to be updated as well?

Reply:
Yes, for consistency I'll update this docstring. Thanks for the catch!